### PR TITLE
Adding admin user to Data 8x hub for Issue: #2813 & Installing R packages raised in #2831

### PR DESCRIPTION
**Pull request by balajialg**

### CODE REVIEW

File: deployments/data8x/config/common.yaml [hunk 1]
```diff
@@ -63,6 +63,9 @@ jupyterhub:
           - b1b001615f9eb7254387f667569e39fb
           - c741d11d7f0bed9930e04276115b1a37
           - fd5ad8c476846392d8dfb1765d1a6ac5
+          # 2813 Eric Van Dusen
+          - ericvd
```
Comment (yuvipanda) [inline deployments/data8x/config/common.yaml]: Their EdX IDs needed to be added here - they look like the random strings you see above. You can find yours by:

1. Enrolling in the class
2. Launching into the lab
3. Finding the user id from control panel

**Review by yuvipanda — CHANGES_REQUESTED** [review]

**Comment by yuvipanda** [comment]
@balajialg can you split this into two PRs? We can't actually merge this one because  the EdX ids are not here yet, and so we can't get the packages installed.

**Comment by yuvipanda** [comment]
You can use the `git cherry-pick` command to get the latest commit alone onto another branch, and use it for a new PR?

**Comment by balajialg** [comment]
@yuvipanda I updated Sean Morris's EdX id as part of this PR and removed the calnet ids! You can check that as part of the [commit ](https://github.com/berkeley-dsep-infra/datahub/pull/2822/commits/a782875ecfc465ec943962dcda6945642740fb42)I made just now. Let me know if you still want me to create a new PR!

Sorry about both the commits! I didn't know how to cherry-pick the last commit and create a different pull request. Thanks for your inputs.

**Pull request merged**